convention-plugins/gradle/libs.versions.toml

```diff
@@ -0,0 +1 @@
+../../gradle/libs.versions.toml
```

Comment:
Using a symlink to access Version Catalog `ln -s gradle/libs.versions.toml convention-plugins/gradle/libs.versions.toml`